app/(authenticated)/(superuser)/admin/page.tsx

```diff
@@ -1,3 +1,20 @@
+import { Button, Card, Stack } from "@mantine/core";
+import Link from "next/link";
+
 export default function AdminPage() {
-  return <p>Hello There!</p>;
+  return (
+    <Card>
+      <Stack>
+        <Link href={"/admin/users"}>
+          <Button variant="default">Manage Users</Button>
+        </Link>
+        <Link href={"/admin/positions"}>
+          <Button variant="default">Manage Positions</Button>
+        </Link>
+        <Link href={"/admin/roles"}>
+          <Button variant="default">Manage Roles</Button>
+        </Link>
```

Comment:
nitpick: `manage` is redundant
```suggestion
        <Link href={"/admin/users"}>
          <Button variant="default">Users</Button>
        </Link>
        <Link href={"/admin/positions"}>
          <Button variant="default">Crew Positions</Button>
        </Link>
        <Link href={"/admin/roles"}>
          <Button variant="default">Roles</Button>
        </Link>
```